utils/loggers/comet/__init__.py

```diff
@@ -0,0 +1,503 @@
+import glob
+import logging
+import os
+
+import comet_ml
+import torch
+import torchvision.transforms as T
+import yaml
+from importlib_metadata import metadata
+
+from utils.dataloaders import img2label_paths
+from utils.general import scale_coords, xywh2xyxy
+from utils.metrics import ConfusionMatrix, box_iou
+
+LOGGER = logging.getLogger(__name__)
+
+COMET_PREFIX = "comet://"
+
+# Project Configuration
+COMET_PROJECT_NAME = os.getenv("COMET_PROJECT_NAME", "yolov5")
```

Comment:
A lot of configuration keys can also be configured in a configuration file. It would be better to use something like:
```
config = comet_ml.config.get_config()
COMET_PROJECT_NAME = config.get_string(USER_VALUE, "comet.project_name", default="yolov5")
```